### PR TITLE
Chrome ext

### PR DESCRIPTION
**Pull request by annezhou920**

### CODE REVIEW

File: src/bookmark-url.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
-	script.src = 'https://forbes.github.io/ad-inspector/src/loader.js';
-	script.id = 'ad-inspector';
-	document.body.appendChild(script);
+(function () {
+	var id = 'ad-inspector';
+	var script = document.getElementById(id);
+	if (!script) {
+		script = document.createElement('script');
+		script.src = 'https://forbes.github.io/ad-inspector/src/loader.js';
+		script.id = 'ad-inspector';
```
Comment (mcscrambles) [inline src/bookmark-url.js]: script.id = id;

File: README.md [hunk 2]
```diff
@@ -2,29 +2,74 @@
...
 This tool should work on all pages that utilize GPT Tags. It does not require any supplemental data to be provided in the page.
 
 ## Contributors
-
-**Anthony Giallella** - ajg723@gmail.com
-**Johnny McCampbell** - jmccampbell@forbes.com
-**Alexander Shnayderman** - ashnayderman@forbes.com
+* **Anthony Giallella** - ajg723@gmail.com
+* **Johnny McCampbell** - jmccampbell@forbes.com
+* **Johnny McCampbell** - jmccampbell@forbes.com
```
Comment (mcscrambles) [inline README.md]: a name so nice, you gotta say it twice ;)

let's remove the second entry and add `**Ben Harrigan** - bharrigan@forbes.com` here

File: README.md [hunk 3]
```diff
@@ -2,29 +2,74 @@
...
+To start the server:
+`npm run start`
 
-## Additional Information
+Assuming you have no errors, you should be able to see your files on your local host (http://127.0.0.1:8080/).
+
+See proxied changes at https://forbes.github.io/ad-inspector/src/loader.js.
+
+## Charles Setup
+This used to be a straight-forward process, but our migration to HTTPS has complicated things.
```
Comment (mcscrambles) [inline README.md]: Let's remove this line -- this is some Forbes specific verbiage that no longer needs to be here.

File: README.md [hunk 4]
```diff
@@ -2,29 +2,74 @@
...
+* [Dev Requirements](#dev-requirements)
 * [Installation](#installation)
+* [Development](#development)
+* [Charles Setup](#charles-setup)
 * [Additional Information](#additional-information)
 * [Contributors](#contributors)
 
+## Dev Requirements
+- [Node](https://nodejs.org/en/)
+- [Charles](https://www.charlesproxy.com/)
```
Comment (mcscrambles) [inline README.md]: Perhaps, this says something like `A proxy server like [Charles](https://www.charlesproxy.com/)`

If devs have some other application they are tied into, they could use it instead.

File: README.md [hunk 5]
```diff
@@ -2,29 +2,74 @@
 
 A bookmarklet for monitoring GPT ad units and their associated targeting on web pages.
 
+* [Dev Requirements](#dev-requirements)
 * [Installation](#installation)
+* [Development](#development)
+* [Charles Setup](#charles-setup)
```
Comment (mcscrambles) [inline README.md:8]: I think we should create a separate readme for Charles Setup since it may or may not be used to develop this application

**Review by mcscrambles — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (annezhou920) [reply README.md:8]: maybe that can be in the Wiki and ill link the Wiki link in readme?

File: README.md (hunk 3 above)
Comment (annezhou920) [reply README.md]: gotcha

File: README.md (hunk 4 above)
Comment (annezhou920) [reply README.md]: gotcha

File: README.md (hunk 2 above)
Comment (annezhou920) [reply README.md]: hahaa

**Review by mcscrambles — APPROVED** [review]

**Pull request merged**